app/styles/custom-components/_vlc-document-list.scss

```diff
@@ -78,4 +78,5 @@
 .grid-container {
   display: grid;
   grid-template-columns: repeat(2,1fr);
+  justify-items: start;
```

Comment:
@brenner-company Deze change heb ik gemaakt omdat anders bij het tonen van de documenten was hetgene dat klikbaar is nogal breed (wat er raar uitzag)

Origineel
![image](https://user-images.githubusercontent.com/22411874/165965413-082efa74-2530-428e-ad7f-131caca8f899.png)

Na changes
![image](https://user-images.githubusercontent.com/22411874/165965502-155daf2f-76b8-4829-b3fb-32d498fa304c.png)

Maar kan natuurlijk ongedaan worden als het eerste de bedoeling was

Reply:
@sergiofenoll Voor mij is dat goed zo 👍 Op zich al beter in lijn met het design dat er aan zit te komen: 

<img width="814" alt="CleanShot 2022-05-02 at 15 44 41@2x" src="https://user-images.githubusercontent.com/18174827/166244145-98cd8eee-d714-4835-8757-657d0a3af5b3.png">